mouse.lisp

```diff
@@ -0,0 +1,137 @@
+;; Copyright (C) 2003-2008 Shawn Betts
+;; Copyright (C) 2017 David Bjergaard
+;;
+;;  This file is part of stumpwm.
+;;
+;; stumpwm is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation; either version 2, or (at your option)
+;; any later version.
+
+;; stumpwm is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this software; see the file COPYING.  If not, see
+;; <http://www.gnu.org/licenses/>.
+
+;; Commentary:
+;;
+;; Functions relating to mouse policies.
+;;
+;; Code:
+(in-package :stumpwm)
+
+(defun temporarilly-disable-sloppy-focus ()
+  "Disable the sloppy pointer for a brief period of time."
+  (unless (or *sloppy-mouse-focus-ignored*
+              (not (eq *mouse-follow-policy* :follow)))
+    (setf *sloppy-mouse-focus-ignored* t)
+    (run-with-timer 0.1 nil #'reenable-sloppy-focus)))
```

Comment:
I'm not keen on having this functionality tied to a timer. I'd prefer if we figure out how handle this with some sort of 'entry'/'exit' hook or `:around` method.